src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -50,6 +59,35 @@ object EntityClient {
     }
   }
 
+  //returns (contents of participants.tsv, contents of samples.tsv)
+  def unzipTSVs(zipFile: ZipFile): (Option[String], Option[String]) = {
+    val zipEntries = zipFile.entries.asScala
+
+    val rand = java.util.UUID.randomUUID.toString.take(8)
+    val participantsFn = s"/tmp/$rand-participants.tsv"
+    val samplesFn = s"/tmp/$rand-samples.tsv"
```

Comment:
Should use `System.getProperty("java.io.tmpdir")` in place of `/tmp`.

Another option is to use `Files.createTempFile`: https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempFile(java.lang.String,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...)